shared/chat/conversation/list/container.js

```diff
@@ -105,6 +106,7 @@ const mapDispatchToProps = (dispatch: Dispatch): DispatchProps => ({
     dispatch(navigateAppend([{props: {message}, selected: 'messageAction'}]))
   },
   onOpenInFileUI: (path: string) => dispatch(({payload: {path}, type: 'fs:openInFileUI'}: OpenInFileUI)),
+  onUsernameClick: (username: string) => dispatch(onUserClick(username)),
```

Comment:
The usernames and avatar are entirely contained in the message wrapper components, which are directly connected so we don't have to create this action here and pass it down. It can be directly created in the wrapper itself. 

If you move the onUsernameClick into this connect it should work and these classes don't need to know about it.
https://github.com/keybase/client/blob/master/shared/chat/conversation/messages/wrapper/container.js 

We have a bunch of older code which connects just at the top and passes the props down and down and down but the approach we're moving towards is to have nested connected components so the only props that need to be sent down are related to the parent.